app/views/whitehall_migration/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Whitehall Migration numero <%= @whitehall_migration.id %> </h1>
+<li>Time started: <%= @whitehall_migration.created_at.strftime("At %T on %m/%d/%Y") %></li>
+<li>Time finished: <%= @whitehall_migration.finished_at.nil? ? "UNFINISHED!!!!!" : @whitehall_migration.finished_at %></li>
+<li>Number of documents: <%= @whitehall_migration.document_imports.count %></li>
+
+<h3>Document Import status</h3>
+
+<% @states.each do |state| %>
```

Comment:
Rather than needing to pass in a variable in for this you can pull it off the document_imports object with `whitehall_migration.document_imports.states` eg:

```
irb(main):009:0> wm.document_imports.states
=> {"pending"=>"pending", "imported"=>"imported", "import_aborted"=>"import_aborted", "import_failed"=>"import_failed", "sync_failed"=>"sync failed", "completed"=>"completed"}
```

Reply:
👍